tests/unit/test_spin.cpp

```diff
@@ -730,7 +730,7 @@ SECTION("Closed-shell spintrace CCD") {
 
 SECTION("Closed-shell spintrace CCSD") {
   auto ctx_resetter = set_scoped_default_context(
-      Context(sequant::mbpt::make_legacy_spaces(/* ignore_spin = */ true),
+      Context(sequant::mbpt::make_legacy_spaces(/* ignore_spin = */ false),
```

Comment:
```suggestion
      Context(sequant::mbpt::make_legacy_spaces(),
```
The default is `false`